android/src/main/java/com/bitmovin/player/reactnative/offline/OfflineDownloadRequest.kt

```diff
@@ -0,0 +1,5 @@
+package com.bitmovin.player.reactnative.offline
+
+class OfflineDownloadRequest(val minimumBitrate: Int, val audioOptionIds: List<String?>?, val textOptionIds: List<String?>?) {
+
+}
```

Comment:
```suggestion
data class OfflineDownloadRequest(
    val minimumBitrate: Int, 
    val audioOptionIds: List<String?>?, 
    val textOptionIds: List<String?>?
)
```

Also, what about the nullablility. I see that the lists might be null, but I don't think we should allow for null entries in the lists 🤔 
(cc @rolandkakonyi )